eoslive-kolibrihome-setup.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Setup Kolibri to use data from Endless Live USB Storage
+After=systemd-tmpfiles-setup.service
+ConditionKernelCommandLine=endless.image.device
+ConditionKernelCommandLine=endless.live_boot
+ConditionDirectoryNotEmpty=/run/kolibrihome
+
+[Service]
+Type=oneshot
+ExecStart=/bin/bash -c 'echo "KOLIBRI_HOME=/run/kolibrihome" > /run/flatpak/overrides/org.learningequality.Kolibri'
```

Comment:
Are we sure this is the correct format for flatpak overrides files? I would expect it to be:

```ini
[Environment]
KOLIBRI_HOME=/run/kolibrihome
```

Reply:
Or invoke `flatpak override`?

Reply:
I think you are right here, I'll fix it for the next revision.

Reply:
Unfortunately `flatpak override` does not have a way to write to `/run`.

Perhaps:

```ini
[Service]
Type=oneshot
ExecStart=cat
StandardInput=data
StandardInputText=[Environment]
StandardInputText=KOLIBRI_HOME=/run/kolibrihome
StandardOutput=file:/run/flatpak/overrides/org.learningequality.Kolibri
```

Reply:
Or a file on disk and a symlink?

Reply:
`echo -e "[Environment]\nKOLIBRI_HOME=/run/kolibrihome"`?